business_objects/general.py

```diff
@@ -46,15 +51,16 @@ def get_session_lookup(exclude_last_x_seconds: int = 5) -> Dict[str, Dict[str, A
     ]
 
 
-def reset_ctx_token(
-    ctx_token: Any,
-    remove_db: Optional[bool] = False,
-) -> None:
+def reset_ctx_token(ctx_token: Any = None, remove_db: Optional[bool] = False) -> None:
     if remove_db:
         session.remove()
-    session_uuid = ctx_token.var.get()
 
-    request_id_ctx_var.reset(ctx_token)
+    session_uuid = request_id_ctx_var.get()
```

Comment:
i think for debugging purposses it would be good to include a print if the session_uuid is None, this would indicate an error somewhere in the app